### DISCUSSION TITLE
SFTP能否增加一个总进度条？

### DISCUSSION THREAD

**Discussion by canonnk** [Eugeny/tabby#6949] [General]
通过 SFTP 上传文件时可以添加总进度条吗？ 如果一次上传的东西太多，不知道是否全部上传完毕，只能向下滚动查看。

Can I add a total progress bar when uploading files via SFTP? If you upload too many things at one time and don't know if they are all uploaded, you can only scroll down to see.

**Comment by lawweiliang** [comment]
@canonnk 
English only please.
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)
